frontend/src/features/dsComponents/body/Body.tsx

```diff
@@ -52,14 +52,15 @@ const extractColumnHeaders = (structure: Structure, value: any[]): ColumnPropert
 const Body: React.FunctionComponent<BodyProps> = ({
   data,
 }) => {
-  const dispatch = useDispatch()
+  // const dispatch = useDispatch()
   const { body, structure } = data
-  const { path, name, peername: username } = data
+  // const { path, name, peername: username } = data
 
   // list out dependencies on dataset body individually for proper memoization
-  useEffect(() => {
-    dispatch(loadBody(newQriRef({ path, name, username }), 1, 100))
-  }, [dispatch, path, name, username])
+  // TODO(chriswhong): reinstate when we want full body UI
+  // useEffect(() => {
+    // dispatch(loadBody(newQriRef({ path, name, username }), 1, 100))
+  // }, [dispatch, path, name, username])
```

Comment:
Instead of commenting this out, add a `preview?: boolean` to `BodyProps`, default it to false & pass `preview` to this effect:

```tsx
useEffect(() => {
  if (preview) { return }
  dispatch(loadBody(newQriRef({ path, name, username }), 1, 100))
}), [dispatch, path, name, username, preview])
```

generally, the `preview` prop should be used to flip this component's behaviour